src/core_landice/mode_forward/mpas_li_calving.F

```diff
@@ -1170,6 +1174,7 @@ subroutine eigencalving(domain, err)
 
          calvingVelocity(:) = 0.0_RKIND
          ! First calculate the front retreat rate (Levermann eq. 1)
+         !calvingVelocity(:) = eigencalvingParameter(:) * max(0.0_RKIND, eMax(:)) * max(0.0_RKIND, eMin(:)) ! m/s
```

Comment:
I don't understand the diff shown here, but double check that the eigencalving calculation of calvingVelocity matches what is in landice/develop here.

Reply:
This change was to make eigencalving work for grounded ice. This PR has eigencalving completely overridden by a uniform scalar calving rate in order to test the apply_calving_velocity routine. I will change this back to normal eigencalving (i.e., calculated from eMax and eMin, only for floating ice).

Reply:
I think it's ok to change it to apply to grounded ice as well.  If so, add that in a separate commit.  The main thing is to avoid leaving it commented out when this PR gets merged.

Reply:
Added in a separate commit